src/data-browser/maps/MapContainer.jsx

```diff
@@ -420,7 +419,14 @@ const MapContainer = props => {
       />
       <h3>Step 2: Select a Variable</h3>
       <p>
-        Then select a variable with the next dropdown
+        Narrow down your selection by filtering on up to two{' '}
```

Comment:
I'm not sure this language really works here, since in this dropdown you can only select one variable. whereas in the dataset filtering view, you use that dropdown to actually select up to two variables.